### PR TITLE
Update ZSH completion for brew 1.19.1

### LINKED ISSUE #4: Update ZSH completion for brew
Correspond with #3

### PR DESCRIPTION
**Pull request by liranmauda**
Update ZSH completion for brew 1.19.1

- Need to be tested (could not run any zsh environment with completion, so this PR is not tested, that why i have marked it as WIP).

Limitations: 
- Currently ignoring the short flags (except for the -h)

 Fixes: #4

### CODE REVIEW

File: zsh/_brew-koji [hunk 1]
```diff
@@ -43,482 +121,1268 @@ _brew-koji() {
...
     (args)
       case $words[1] in
+       (add-external-repo)
+         _arguments -A "-*" \
+            '(--tag --priority --mode )'{-h,--help}'[show this help message and exit]' \
+            '--tag=-[Also add repo to tag. Use tag::N to set priority]' \ 
+            '--priority=-[Set priority (when adding to tag)]' \ 
+            '--mode=-[Set merge mode]' \ 
+            '*: name [url]' \ 
+            && ret=0
```
Comment (rnc) [inline zsh/_brew-koji]: I think the trailing whitespace causes an issue. Can we remove all trailing please.

File: zsh/_brew-koji [hunk 2]
```diff
@@ -43,482 +121,1268 @@ _brew-koji() {
...
+            '(--new )'{-h,--help}'[show this help message and exit]' \
+            '--new[Create a new permission]' \ 
+            '*:<permission> <user> [<user> ...]' \ 
+            && ret=0 
+            ;;
+       (import)
+         _arguments -A "-*" \
+            '(--link --test --create-build --src-epoch )'{-h,--help}'[show this help message and exit]' \
+            '--link[Attempt to hardlink instead of uploading]' \ 
+            '--test[Don't actually import]' \
```
Comment (rnc) [inline zsh/_brew-koji]: The `Don't` causes quoting issues. I think in this causes its better to change all instances to

```
"--test[Don't actually import]" \
```

Note that there are also a few instances of `Dont` that could be fixed up as well.

File: zsh/_brew-koji [hunk 3]
```diff
@@ -43,482 +121,1268 @@ _brew-koji() {
...
+            '--days=-[Timeout before clearing]' \ 
+            '--pkg=-[Limit to a single package]' \ 
+            '--build=-[Limit to a single build]' \ 
+            '--ignore-tag=-[Ignore these tags when considering whether a build is/was latest]' \ 
+            '--ignore-tag-file=-[File to read tag ignore patterns from]' \ 
+            '--protect-tag=-[Do not prune signed copies from matching tags]' \ 
+            '--protect-tag-file=-[File to read tag protect patterns from]' \ 
+            '--trashcan-tag=-[Specify trashcan tag]' \ 
+            '--debug[Show debugging output]' \ 
+            '*: \'
```
Comment (rnc) [inline zsh/_brew-koji]: Incorrect layout.

```suggestion
            '*:' \
```

**Review by rnc — CHANGES_REQUESTED** [review]

File: zsh/_brew-koji (hunk 2 above)
Comment (liranmauda) [reply zsh/_brew-koji]: Fixed all `Don't` instances:

$ grep -i "dont" brew-koji/zsh/_brew-koji 
$ grep -i "don't" brew-koji/zsh/_brew-koji
            "--test[Don't actually import]" \
            "--test[Don't actually import]" \
            "--test[Test mode -- don't actually import]" \
            "--nowait[Don't wait on for regen to finish]" \
            "--nowait[Don't wait on task]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on image creation]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on task]" \
            "--nowait[Don't wait on appliance creation]" \
            "--nowait[Don't wait on livecd creation]" \
            "--nowait[Don't wait on livemedia creation]" \
            "--nowait[Don't wait on build]" \
            "--skip-tag[If creating a new build, don't tag it]" \
            "--nowait[Don't wait on build]" \
            "--noinherit[Don't follow inheritance]" \

**Review by rnc — APPROVED** [review]
Thanks! Noticed another `'s` issue but I'll fix that up now.

**Pull request merged**